pkg/tracker/disputeChecker_test.go

```diff
@@ -43,8 +45,9 @@ func TestDisputeCheckerOutOfRange(t *testing.T) {
 	execEthUsdPsrs(ctx, t, ethUSDPairs)
 	time.Sleep(2 * time.Second)
 	execEthUsdPsrs(ctx, t, ethUSDPairs)
+	ctx = context.WithValue(ctx, tellorCommon.ContractAddress, common.Address{0x0000000000000000000000000000000000000000})
```

Comment:
I added this in `testutil.CreateContext(t)` so shouldn't be needed

Reply:
Ok! I'll remove it.